### PR TITLE
remove opencensus docs

### LINKED ISSUE #15086: investigate opencensusagent doc tests (remove?)
The tests were broken when we bumped master back to istio/istio master. Attempted to fix through https://github.com/istio/istio.io/pull/15084 without much luck, so the test was disabled to unblock master.

We need to fix the test. In short, it appears that the sidecar does not start in the test which causes bookinfo to timeout on the deployment.

### PR DESCRIPTION
**Pull request by dhawton**
## Description

Remove OpenCensus Agent docs as it is being removed.

Closes #15086 

## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: content/en/about/faq/distributed-tracing/how-to-support-tracing.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ If you are using Lightstep, you will also need to forward the following headers:
 
 - `x-ot-span-context`
 
-If you are using OpenTelemetry, OpenCensus or Stackdriver, you will also need to forward the following headers:
+If you are using OpenTelemetry or Stackdriver, you will also need to forward the following headers:
```
Comment (kfaseela) [inline content/en/about/faq/distributed-tracing/how-to-support-tracing.md:22]: what about Stackdriver? I saw some PRs removing stackdriver as well?

Comment (dhawton) [reply content/en/about/faq/distributed-tracing/how-to-support-tracing.md:22]: Good call.. let me double check

Comment (kfaseela) [reply content/en/about/faq/distributed-tracing/how-to-support-tracing.md:22]: otherwise LGTM

**Review by kfaseela — APPROVED** [review]

File: content/en/about/faq/distributed-tracing/how-to-support-tracing.md (hunk 1 above)
Comment (dhawton) [reply content/en/about/faq/distributed-tracing/how-to-support-tracing.md:22]: I can only find tests being removed and comments from last week that it *will* be removed (future tense). So I think we should do Stackdriver removals when it is removed.

Comment (kfaseela) [reply content/en/about/faq/distributed-tracing/how-to-support-tracing.md:22]: sounds good

Comment (howardjohn) [reply content/en/about/faq/distributed-tracing/how-to-support-tracing.md:22]: fyi @kyessenov

**Pull request merged**